main.cpp

```diff
@@ -59,63 +65,126 @@ video: Chapter 2 - Part 3
 void variableDeclarations()
 {
     //example:
-    int number = 2; //declaration of a variable named "number", that uses the primitive type 'int', and the variable's initial value is '2'
-    
-
-    
-    ignoreUnused(number); //passing each variable declared to the ignoreUnused() function
+    int number = 2; //declaration of a variable named "number", that uses the primitive type 'int', and the variable's initial value is '2'
+
+    int numbers = 0;
+    int threshold = 2;
+    int ratio = 4;
+
+    float height = 1.3f;
+    float width = 1.8f;
+    float length = 1.9f;
+
+    double reverbTime = 1.34;
+    double reverbLength = 1.346;
+    double reverbEr = 1.566;
+
+    bool distortion = false;
+    bool delay = true;
+    bool equalistion = true;
+
+    char initial = 'a';
+    char letter = 'b';
+    char alpha = 'c';
+
+    ignoreUnused (number, numbers, threshold, ratio, height, width, length, reverbTime,reverbLength, reverbEr, distortion, delay, equalistion, initial, letter, alpha);
+

```

Comment:
Fix the formatting here.   Your work needs to conform to the course coding standard
- no spaces around `(` and `)`
You need to train your eye to see code that is formatted differently from the coding standard being used in whatever project or team you're working with.
- [ ] revise